src/components/WwwFrame/TheHeader.vue

```diff
@@ -77,10 +77,6 @@
 							data-testid="header-log-in"
 							:event="showPopupLogin ? '' : 'click'"
 							@click.native="auth0Login"
-							v-kv-track-event="[
-								['TopNav','click-Sign-in'],
```

Comment:
We need to preserve this simple `click-Sign-in` event in both instances. It can now be: `v-kv-track-event="['TopNav','click-Sign-in']"`

Reply:
ok